portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Causes the bubble to slide down the page.
+function bubbleFall() {
+    // Find the bubble.
+    const elem = document.getElementById('bubble');
+    let pos = 0;
+    // Make the movement visible and fluid by moving every 10 milliseconds;
+    const id = setInterval(frame, 10);
+    // For each movement, move the element one pixel down. At the bottom, stop moving.
+    function frame() {
```

Comment:
```suggestion
    function lowerBubbleOnePixel() {
```
The more specific a function name is the better. When I read `setInterval(frame, 10);` I can't understand that code without reading `frame()` to see what it does. If i saw `setInterval(lowerBubbleOnePixel, 10);` I'd assume that we're lowering the bubble by one pixel every 10 seconds.